_includes/header.html

```diff
@@ -1,7 +1,7 @@
 <header>
   <div class="header__branding">
-    <img class="header__logo" src="img/harrow_woodcraft_logo.png" alt="The Harrow Woodcraft logo">
-    <h1 class="header__headline">HARROW WOODCRAFT FOLK</h1>
+   <a href="http://www.harrowwoodcraftfolk.btck.co.uk"><img class="header__logo" src="img/harrow_woodcraft_logo.png" alt="The Harrow Woodcraft logo"></a>
```

Comment:
great linkage here - I wonder if we should be linking to our home page rather than the other site?

Reply:
As the same header.html is also already used by the home page, we thought to link the logo to the main harrow woodcraft site.